docs/architecture/accounts.md

```diff
@@ -21,28 +21,27 @@ In Miden every account is a smart contract. The diagram below illustrates the ba
 
 ### Account ID
 
-!!! info
-    Its `~63` bits long identifier for the account id (one field element). 
+A `~63` bits long identifier for the account ID (one field element `felt`). 
 
-The four most significant bits specify the [account type](#account-types) - regular, immutable, faucet - and the [storage mode](#account-storage-modes) - public or private.
+The four most significant bits specify the [account type](#account-types) - regular or faucet - and the [storage mode](#account-storage-modes) - public or private.
 
-### Account storage
+### Account data storage
 
-Account storage for user-defined data is composed of two components.
+Storage for user-defined data. It is composed of two components.
```

Comment:
```suggestion
A storage for user-defined data that is composed of two components.
```

Reply:
I would keep the articles for this sort of documentation. For API docs, we remove them.

Reply:
ok, let's do that then